ui/src/browserStore.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright © 2021 The Radicle Upstream Contributors
+//
+// This file is part of radicle-upstream, distributed under the GPLv3
+// with Radicle Linking Exception. For full terms see the included
+// LICENSE file.
+
+import type * as zod from "zod";
+import persistentStore from "svelte-persistent-store/dist";
+
+import * as svelteStore from "ui/src/svelteStore";
+import * as error from "ui/src/error";
+
+const KEY_RE = /radicle(\.[a-zA-Z])+/;
```

Comment:
Could we call this `VALID_STORAGE_KEY_MATCH`?
We have similar constants elsewhere: `VALID_SEED_MATCH`, `VALID_PEER_MATCH`.